src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-dialog.tsx

```diff
@@ -11,11 +11,12 @@ import {
     Equipment,
     EquipmentInfos,
     EquipmentItem,
+    EquipmentType,
     equipmentStyles,
     useSnackMessage,
     // Equipment,
```

Comment:
not in your code , but you can remove the unused code
```suggestion
```

Reply:
removed